### DISCUSSION TITLE
opening TLS error

### DISCUSSION THREAD

**Discussion by yeahse** [planetscale/discussion#319] [Bug Reports] [answered]
locally it works fine "localhost:8080"

but when I uploaded it to Cloud Run (GCP)

it shows 

 25 const test = yield prisma_1.prisma.post.findMany(
2022-10-26 14:31:34.037 JST
Error opening a TLS connection: error:16000069:STORE routines:func(0):unregistered scheme:../deps/openssl/openssl/crypto/store/store_register.c:237:scheme=file, error:80000002:system library:func(0):reason(2):../deps/openssl/openssl/providers/implementations/storemgmt/file_store.c:267:calling stat(/home/iojs/build/ws/out/Release/obj.target/deps/openssl/certs), error:16000069:STORE routines:func(0):unregistered scheme:../deps/openssl/openssl/crypto/store/store_register.c:237:scheme=file, error:80000002:system library:func(0):reason(2):../deps/openssl/openssl/providers/implementations/storemgmt/file_store.c:267:calling stat(/home/iojs/build/ws/out/Release/obj.target/deps/openssl/certs), error:16000069:STORE routines:func(0):unregistered scheme:../deps/openssl/openssl/crypto/store/store_register.c:237:scheme=file, error:80000002:system library:func(0):reason(2):../deps/openssl/openssl/providers/implementations/storemgmt/file_store.c:267:calling stat(/home/iojs/build/ws/out/Release/obj.target/deps/openssl/certs), error:16000069:STORE routines:func(0):unregistered scheme:../deps/openssl/openssl/crypto/store/store_register.c:237:scheme=file, error:80000002:system library:func(0):reason(2):../deps/openssl/openssl/providers/implementations/storemgmt/file_store.c:267:calling stat(/home/iojs/build/ws/out/Release/obj.target/deps/openssl/certs), error:0A000086:SSL routines:func(0):certificate verify failed:../deps/openssl/openssl/ssl/statem/statem_clnt.c:1896: (unable to get local issuer certificate)

 what is the problem??

database url is 

url = "mysql://{id}:{pscale_pw}@ap-northeast.connect.psdb.cloud/tst?sslaccept=strict"

**Comment by fooforge** [comment, marked as answer]
Hi @yeahse,

This error occurs if the server is unable to validate PlanetScale's SSL/TLS certificate. It is necessary to be able to connect to PlanetScale securely. We explain the concepts [in our documentation](https://planetscale.com/docs/concepts/secure-connections) and I would like to encourage you to read through it all.

In a nut-shell, you will need to have the certificate authorities' root certificates installed on your container or server which is usually achieved by installing the `ca-certificates` package. Some database drivers have auto-discovery capabilities for the CA root certificates on your system, others you will need to configure to use the correct file. We have instructions for the most common Linux, *nix and Windows flavors lined out in the above linked document.
